sources/osgShader/ShaderProgramBuilder.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
```

Comment:
Program would become osgShader, (optimizer, preprocessor) dependant.

ShaderProgramBuilder  is at the corner of Shader, Shadergenerator/Compiler, Composer and ShaderPreprocessor/preprocessor/optimizer
(imho it should do the shaderprocessor thing too)